src/tracing/errors.ts

```diff
@@ -4,21 +4,25 @@ import { ClientOptions, ErrorEvent, EventHint } from '@sentry/types'
 function isEthersRequestError(error: Error): error is Error & { requestBody: string } {
   return 'requestBody' in error && typeof (error as unknown as Record<'requestBody', unknown>).requestBody === 'string'
 }
-
 /**
+
  * Filters known (ignorable) errors out before sending them to Sentry.
  * Intended as a {@link ClientOptions.beforeSend} callback. Returning null filters the error from Sentry.
  */
 export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event: ErrorEvent, hint: EventHint) => {
   const error = hint.originalException
   if (error instanceof Error) {
-    // ethers aggressively polls for block number, and it sometimes fails (whether spuriously or through rate-limiting).
-    // If block number polling, it should not be considered an exception.
     if (isEthersRequestError(error)) {
       const method = JSON.parse(error.requestBody).method
+      // ethers aggressively polls for block number, and it sometimes fails (whether spuriously or through rate-limiting).
+      // If block number polling, it should not be considered an exception.
       if (method === 'eth_blockNumber') return null
     }
 
+    // for now, these errors are not actionable so we should not report them.
```

Comment:
unclear if we need this to live within the `isEthersRequestError` section above?

Reply:
I think this should match an ethers request error, so you can include it in that section to increase specificity of the matching.